ui/app/(ai)/analyst/(tools)/scans.ts

```diff
@@ -0,0 +1,30 @@
+import { tool } from "@langchain/core/tools";
+
+import { getScan, getScans } from "@/actions/scans";
+import { getScanSchema, getScansSchema } from "@/types/lighthouse";
+
+export const getScansTool = tool(
```

Comment:
Following up on this comment: https://github.com/prowler-cloud/prowler/pull/7508#discussion_r2052086824 

Could we please use the `/app` folder only for routing purposes?

Thank you!

Reply:
Moved all tools code to `/lib/lighthouse/tools/` directory